### PR TITLE
replace findOne + orderBy to find + sort + limit

### PR DESCRIPTION
**Pull request by AlvaroVega**
https://github.com/telefonicaid/perseo-fe/issues/339

### CODE REVIEW

**Comment by AlvaroVega** [comment]
Following tests are in error:
      1) should serialize actions, executing only one when there is an elapsed condition, even when arriving "together"
      2) should update execution time when an action with interval has been executed
      3) should not execute an action when has been executed too recently
      4) should return silent entities

**Comment by AlvaroVega** [comment]
To be merged with squash

File: lib/models/executionsStore.js [hunk 1]
```diff
@@ -42,19 +42,27 @@ module.exports = {
                 myutils.logErrorIf(err);
```
Comment (fgalan) [inline lib/models/executionsStore.js]: CHANGES_NEXT_RELEASE entry missing

Comment (AlvaroVega) [reply lib/models/executionsStore.js]: fixed in https://github.com/telefonicaid/perseo-fe/pull/340/commits/13388f37c7ab09c1b5f0d156d632aa7cd8df5ca0

**Pull request merged**